ZupIT/ritchie-cli
# Add release-v3 workflow

Signed-off-by: GuillaumeFalourd <guillaume.falourd@zup.com.br>

### Description

- Add release-v3 workflow without release publication on the repository (yet). Just to add binaries on AWS S3 bucket.

### How to verify it

- Execute the workflow and check if the files are available on the AWS S3 bucket.

### Changelog

- Add release-v3 workflow

### Tasks

- [x] Create release workflow for v3.
- [x] Update install.sh script file.
- [x] Add new bucket secret for Ritchie v3.
- [x] Update AWS S3 bucket URL in `install.sh` script.


---------------------------------------------


### This pull request generated the following artifacts.
To test the health and quality of this implementation, download the respective binary for your operating system, unzip and directly run the binary like the examples below.
- **Windows**
 Download the file: **[rit-windows.zip](https://github.com/ZupIT/ritchie-cli/suites/3738171263/artifacts/91627976)**
 Unzip to some folder like: `C:\home\user\downloads\pr1031`
 Access the folder: `cd C:\home\user\downloads\pr1031`
 Directly call the binary: `.\rit.exe --version` or `.\rit.exe name of formula`
 

 - **Linux**
 Download the file: **[rit-linux.zip](https://github.com/ZupIT/ritchie-cli/suites/3738171263/artifacts/91627974)**
 Unzip to some folder like: `/home/user/downloads/pr1031`
 Access the folder: `cd /home/user/downloads/pr1031`
 Assign execute permission to binary: `chmod +x ./rit`
 Directly call the binary: `./rit --version` or `./rit name of formula`
 

 - **MacOS**
 Download the file: **[rit-macos.zip](https://github.com/ZupIT/ritchie-cli/suites/3738171263/artifacts/91627975)**
 Unzip to some folder like: `/home/user/downloads/pr1031`
 Access the folder: `cd /home/user/downloads/pr1031`
 Assign execute permission to binary: `chmod +x ./rit`
 Directly call the binary: `./rit --version` or `./rit name of formula`

> Generated at Fri Sep 10 2021 19:49:26 GMT+0000 (Coordinated Universal Time)
